### ISSUE TITLE
rebuild done, just need data

### ISSUE THREAD

**Issue by katieannemills** [GRIFFINCollaboration/CycleAssistant#3]
new UI is up for preview here: http://griffincollaboration.github.io/CycleAssistant

but, I understand @SmithJK has some matrix magic to help include daughter activity in the activity estimates; feel free to just PR the C into this repo, and I'll see if we can't turn it into something client side.

**Comment by SmithJK** [comment]
So done. Thanks for the reminder.

**Comment by katieannemills** [comment]
Great, thanks! But - just looking at the code briefly here, this looks like it's only considering one beam cycle; I'm looking at lines 159-160, where the maximum time considered appears to be one implantation period + one decay period. What if I want to know activities one week into an experiment with 1s cycles? I've yet to see a way to determine this that doesn't involve some pretty massive computations, when decay products are included.

It may behoove us to back up a step here - the point of this exercise (as I understood it) was to make sensible activity estimates for things like safety reports. What is the current standard method for these calculations?

**Comment by SmithJK** [comment]
Yes, that program was designed with a slightly different aim: not so much the safety reports or accumulated activity over a week, but optimizing the science output based on cycle times for beam on and beam off.
